client/src/app/app.component.ts

```diff
@@ -114,6 +114,8 @@ export class AppComponent implements OnInit, AfterViewInit {
     let resetScroll = true
     const eventsObs = this.router.events
 
+    this.viewportScroller.setOffset([0, 81]) // sub-menu-height
```

Comment:
Should only be applied if targeting `my-account` no?

Reply:
`my-account` and `about` ? Maybe we would add any anchors in `about/instance`

Reply:
yes